spec/features/filling_in_the_form/fill_in_the_form_spec.rb

```diff
@@ -5,7 +5,7 @@
 RSpec.feature "Fill in the form" do
   include FillInTheFormSteps
 
-  scenario "Complete the form" do
+  scenario "Complete the form", js: true do
```

Comment:
Do we want to have two of these: one with JS and one without?

Reply:
Thanks for the review @issyl0 and your comments - with and without js tests added.